shop/shop/src/main/resources/db/migration/V2__schema.sql

```diff
@@ -0,0 +1 @@
+CREATE SCHEMA IF NOT EXISTS shop;
```

Comment:
The schema creation should be the first step in the migrations. I would also suggest to join the table creations in only one migration called `tables_initialization` or something similar because it does not really make sense to have 9 separate files for this